README.md

```diff
@@ -46,6 +46,10 @@ If you want to use a package manager:
 - [Snap](https://snapcraft.io/) users can use `sudo snap install helm
   --classic`.
 
+If you want to install helm from script:
+
+- [Script](https://github.com/helm/helm/blob/master/docs/install.md#from-script) instructions to install from script on a new instance without Helm. For example, you want to install on a container, cluster, etc. Note: GCP console comes with `Helm` installed by default.
```

Comment:
can we reference the official docs instead, which is built from the docs folder? https://helm.sh/docs/using_helm/#from-script

Reply:
Also, under "GCP console comes with `Helm` installed", `Helm` should be lowercase. Since it's wrapped in backticks, it implies it's referencing the name used in the console which is lowercase.